Mathlib/NumberTheory/NumberField/InfinitePlace/TotallyRealComplex.lean

```diff
@@ -146,12 +146,18 @@ A number field `K` is totally complex if all of its infinite places are complex.
 @[mk_iff] class IsTotallyComplex (K : Type*) [Field K] [NumberField K] where
   isComplex : ∀ v : InfinitePlace K, v.IsComplex
 
-variable {K : Type*} [Field K] [NumberField K]
+variable {F : Type*} [Field F] {K : Type*} [Field K] [NumberField K] [Algebra F K]
 
 theorem nrRealPlaces_eq_zero_iff :
     nrRealPlaces K = 0 ↔ IsTotallyComplex K := by
   simp [Fintype.card_eq_zero_iff, isEmpty_subtype, isTotallyComplex_iff]
 
+theorem IsTotallyComplex.isConj_ne_one [IsTotallyComplex K] {φ : K →+* ℂ} {σ : K ≃ₐ[F] K}
```

Comment:
Do you really need `K` to be *totally* complex for this result? I don't feel so (and inspecting the proof it does not look like so neither...)

Reply:
Indeed, you're right. I added `ComplexEmbedding.isConj_ne_one_iff` for the general case, but it still think it is useful to have the specialized statement for totally complex fields

Reply:
Well, but then do you still need the assumption `IsTotallyComplex K`?

Reply:
Yes, it's the reverse direction of the new result `isConj_ne_one_iff`